docs/wiki/framework/events-framework.md

```diff
@@ -110,9 +110,10 @@ MenuType: 0 = Interaction, 1 = Self Interaction
 
 | Event Key | Parameters | Locality | Type | Description |
 |----------|---------|---------|---------|---------|---------|
-|`ace_cookoff_cookOff` | _vehicle | Global | Listen | Vehicle cook off has started
-|`ace_cookoff_cookOffBox` | _box | Global | Listen | Ammo box cook off has started
-|`ace_cookoff_engineFire` | _vehicle | Global | Listen | Engine fire has started
+|`ace_cookoff_cookOff` | [_vehicle, _intensity, _instigator, _smokeDelayEnabled, _ammoDetonationChance, _detonateAfterCookoff, _fireSource, _canRing, _maxIntensity, _canJet] | Server | Callable | Start vehicle cook-off
+|`ace_cookoff_cookOffBox` | [_box, _killer, _instigator] | Server | Callable | Start ammo box cook-off
+|`ace_cookoff_detonateAmmunition` | [_object, _magazines, _totalAmmo, _destroyWhenFinished, _killer, _instigator, _initialDelay"] | Server | Callable | Start ammo detonation
+|`ace_cookoff_engineFire` | _vehicle | Server | Callable | Start engine fire
```

Comment:
I'm unsure about what to do: I'm not sure if these events should be public or not.

Reply:
Fine as they are IMO